lib/ui/views/pool_list/layouts/components/pool_remove_favorite_icon.dart

```diff
@@ -121,14 +119,20 @@ class PoolRemoveFavoriteIcon extends ConsumerWidget {
                                           )!
                                               .yes,
                                           onPressed: () async {
-                                            ref.read(
-                                              DexPoolProviders
-                                                  .removePoolFromFavorite(
-                                                poolAddress,
-                                              ),
-                                            );
+                                            ref
+                                              ..read(
+                                                DexPoolProviders
+                                                    .removePoolFromFavorite(
+                                                  poolAddress,
+                                                ),
+                                              )
+                                              ..invalidate(
```

Comment:
I think that `invalidate` should be done in the `removePoolFromFavorite` provider.